_posts/2016-08-11-React-Native-at-Artsy.markdown

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: "React Native at Artsy"
+date: 2016-08-11 21:17
+comments: true
+article-class: "expanded-code"
+categories: [React, Mobile]
+author: eloy
+---
+
+<center>
+<img src="/images/react-native/artsy_react_logo.svg" style="width:300px;">
+</center>
+
+As [the Artsy iOS app](https://github.com/artsy/eigen) grew larger, we came to a point where we started hitting pain
+points in terms of our existing architecture, the future directions the application might take, support for other future
+platforms such as Android, and the technology decisions we needed to take in-order steer this all in the direction 
+we want - including more ways to improve user experience, such as, more efficient network data fetching.
+
+Technology-wise, it would have been an easy leap to start using Swift and, as a matter of fact, we did just that in both
+our flagship application and [an auction kiosk app](http://artsy.github.io/blog/2014/11/13/eidolon-retrospective/).
+However, while Swift is a great language, it’s also just that, _another_ language. It does not do much in terms of new
+paradigms that solved our architectural needs, it does not help in terms of cross-platform as most of our code is
+building views and thus very framework specific, and did not really offer anything in terms of more efficient network
+data fetching.
+
+Our web team have been integrating React throughout their projects, and with really great results. There is a mobile version, 
+and as luck would have it, the solutions provided by the React ecosystem ticked all of our boxes too.
+
+<!-- more -->
+
+# Cons
+
+So you decided to read the full post and the first thing you get confronted with is a list of cons… _boo_. Let’s just get
```

Comment:
So you decided to read the full post and the first thing you *get* confronted with is a list of cons -> *got* and *was*